### DISCUSSION TITLE
Bob 2.4.4 now released!

### DISCUSSION THREAD

**Discussion by edmundreinhardt** [IBM/ibmi-tobi#154] [Announcements]
Hurray we now have full lowercase support and INCDIR support for your include files and we support specifying the target EBCDIC CCSID if you are compiling into different CCSIDs than the job.

We have had simplified Rules.mk where you only need targets like this with no extra baggage.
`OBJECT.PGM: myname.pgm.rpgle `

If you have installed 2.4.0 or later, please upgrade to 2.4.4 as many errors were fixed.

**Comment by JDubbTX** [comment]
Hello Edmund,

On 2.4.4, lowercase doesn't seem to be supported with binder language file names 'srvpgmnm.bnd' didn't work for me but 'SRVPGMNM.BND' did.

**Reply by edmundreinhardt** [reply]
Thanks for bringing this to our attention.  I will create an issue to track this.

**Comment by edmundreinhardt** [comment]
@JDubbTX  I opened an issue to track this
https://github.com/IBM/ibmi-bob/issues/177

However our standard example https://github.com/edmundreinhardt/bob-recursive-example
includes functionsVAT/fvat.bnd which I can build from without any trouble
